examples/features/nse-composition/passthrough-3.yaml

```diff
@@ -3,16 +3,17 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: nse-firewall-vpp
+    app: nse-passthrough-3
   name: nse-passthrough-3
 spec:
   selector:
     matchLabels:
-      app: nse-firewall-vpp
```

Comment:
I don't quite follow why do we change this in this PR.
Please apply fixes for composition examples in the separate PR.

Reply:
ok, done